apps/server/src/modules/authorization/authorization.service.spec.ts

```diff
@@ -207,34 +200,6 @@ describe('authorization.service', () => {
 		});
 	});
 
-	describe('checkPermissionsByReferences', () => {
-		const permissionTrue = 'true' as Permission;
-		const permissionFalse = 'false' as Permission;
-
-		it('should call ReferenceLoader.getUserWithPermissions', async () => {
-			const userId = new ObjectId().toHexString();
-			const entityName = AllowedAuthorizationEntityType.Course;
-			const entityId = new ObjectId().toHexString();
-			const spy = jest.spyOn(service, 'hasPermission');
-			spy.mockImplementation((user: User, entity: PermissionTypes, context: IPermissionContext) => {
-				return context.requiredPermissions[0] === permissionTrue;
-			});
-
-			const retMap = service.hasPermissionsByReferences(
-				userId,
-				entityName,
-				entityId,
-				[permissionTrue, permissionFalse],
-				Actions.read
-			);
-
-			expect(await retMap.get(permissionTrue)).toBe(true);
-			expect(await retMap.get(permissionFalse)).toBe(false);
-
-			spy.mockRestore();
-		});
-	});
-
 	describe('getUserWithPermissions', () => {
 		it('Should call ReferenceLoader.getUserWithPermissions.', async () => {
```

Comment:
Wir haben in den anderen Tests, die wir zuletzt geschrieben haben für jedes Szenario noch ein Describe Block eingefügt. Hier wären die beiden Szenarios wahrscheinlich sowas wie: "Gets user successfully" und "User can not be found". Die Mocks und Parameter haben wir in Setup Funktionen ausgelagert.